src/app/features/settings/notifications/notifications.component.ts

```diff
@@ -125,7 +137,24 @@ export class NotificationsComponent implements OnInit {
   onSubscriptionChange(event: SubscriptionEvent, frequency: SubscriptionFrequency) {
     const user = this.currentUser();
     if (!user) return;
-    const id = `${user.id}_${event}`;
+
+    const eventKey = event as unknown as string;
+
+    const apiEventName = this.FORM_EVENT_TO_API_EVENT[eventKey] ?? eventKey;
+
+    let id: string | undefined;
+
+    if (event === SubscriptionEvent.GlobalReviews) {
+      const subs = this.notificationSubscriptions();
+      const match = subs.find((s) => (s.event as string) === 'new_pending_submissions');
+      if (match) {
+        id = match.id;
+      } else {
+        return;
+      }
+    } else {
+      id = `${user.id}_${apiEventName}`;
+    }
```

Comment:
Here’s a simplified version. I’m not sure if it works as expected, but it’s worth a try. 
```suggestion
const apiEventName = this.FORM_EVENT_TO_API_EVENT[event as string] ?? (event as string);

    const id =
      event === SubscriptionEvent.GlobalReviews
        ? this.notificationSubscriptions().find((s) => (s.event as string) === 'new_pending_submissions')?.id
        : `${user.id}_${apiEventName}`;

    if (!id) return;
```

Reply:
I tried this and it causes the dropdowns to break somehow